### ISSUE TITLE
Issuer Only Context messing up completions

### ISSUE THREAD

**Issue by aikar** [aikar/commands#79]
```java
@CommandCompletion("@groupmembers") // TODO: Make ACF support skipping group for completions
    @Syntax("<player> &e- Locate a player in your group using a compass.")
    public static void onGroupMemberLocate(Player player, Group group, OnlinePlayer toTarget) {
```

group members is getting processed for Group, which doesn't need to be completed.

**Comment by DarkPaladinUHC** [comment]
Is this saying that the `@groupmembers` completion is being applied to `group` insread of `toTarget`?

**Comment by aikar** [comment]
@DarkPaladinUHC yes

**Comment by DarkPaladinUHC** [comment]
Any ideas on how to fix this, the best thing I can think of would be like `@CommandCompletion("default @groupmembers")`

**Comment by aikar** [comment]
Issue was actually the issuer and nothing to do with "Issuer Only", as we were not even inspecting the type of param

fixed in 633c6278b576fefb58c2ecf7d4925f656b036514
